joss/paper.md

```diff
@@ -0,0 +1,111 @@
+
+ - `xrtpy.image_correction.deconvolve`: Deconvolution is a powerful technique for improving image sharpness. The `deconvolve` function applies deconvolution to XRT images, effectively reducing blurring effects caused by the telescope's point spread function.
+
+ - `xrtpy.image_correction.remove_lightleak.py`: The `remove_lightleak` function in this module eliminates light leak (visible stray light) from XRT synoptic composite images. This results in cleaner and more precise images suitable for in-depth analysis.
+
+XRTpy supports multiple elemental abundance sets, including CHIANTI coronal abundances ([@Feldman:1992], [@Feldman:2014]) , hybrid abundances (based on [Fludra-and-Schmelz:1999] and  [@Schmelz:2012]), and photospheric abundances (based on [@Grevesse:2007], [@Scott:2015], and [@Asplund:2009]).
+
+Researchers have the flexibility to select the abundance model that best aligns with their research requirements.
+
+
+XRTpy's capabilities are designed to empower researchers and scientists to fully exploit the potential of XRT data, offering the scientific community a unique opportunity to study the Sun's dynamic and complex behavior in a user-friendly and efficient manner.
+
+
+![Figure 1: The Effective area for all XRT filters plotted using XRTpy. .\label{fig:Figure 1}](xrtpy_effective_area_plot.pdf)
+
+![Figure 2: The temperature response is plotted for all XRT filters using XRTpy.  The plot also shows the effects of using different abundance models from Chianti for each filter.. .\label{fig:Figure 2}](xrtpy_temperature_response_plot.pdf)
+
+![In Figure 3, the application of the `temperature_from_filter_ratio` function is illustrated, demonstrating its role in calculating electron temperature and volume emission measure through filter ratios. The dataset, collected on January 28, 2011, between 01:31:55 and 01:32:05 UTC, comprises two images captured with specific filters. These images offer unique insights into solar conditions during the observed moments, as shown by [@Guidoni:2015]. .\label{fig:Figure 3}](xrtpy_temperature_from_filter_ratio_plot.pdf)
+
+# Development of XRTpy Version 0.4.0
+
+XRTpy version 0.4.0, released on December 5, 2023, represents a significant tool in solar physics data analysis. This version, available through the Python Package Index (PyPI), can be easily installed using pip and is compatible with Python 3.11 and later, aligning with the latest Python standards.
+
+In fostering collaboration within the solar physics community, XRTpy seamlessly integrates with similar packages. Designed towards working in conjunction with aiapy for SDO/AIA observations, EISpack for Hinode-EIS data analysis, and irispy-lmsal for IRIS observations. This integration provides users with a smooth and comprehensive analysis experience. Further building on the SunPy framework (SunPy Project et al., 2020), XRTpy effectively utilizes the `Map` object for handling Hinode-XRT image data.
```

Comment:
I'm wondering here about including the word "seamless".  Would it make sense to describe the interoperability with other packages as a work in progress, and that we've started on this with the developers of aiapy?

Reply:
Ah, yes!
I was having a hard time thinking of the right word, thanks!